doc/Grammalecte.txt

```diff
@@ -48,8 +48,8 @@ Or on github:
   https://github.com/dpelle/vim-Grammalecte
 
 Grammalecte can be downloaded at:
-
-  http://grammalecte.net/telecharger.php
+ 
+  http://www.dicollecte.org/grammalecte/telecharger.php
```

Comment:
Both URL work for me, but since it grammalecte.net redirects to www.dicollecte.org, I'll merge the pull request.